### PR TITLE
Feature/faqs styling mobile

### PR DESCRIPTION
**Pull request by zacktraczyk**
Problem
=======
- Add mobile styling for FAQs

Solution
========
- Added star icon and styled below 600px

Change Summary:
---------------
-

### CODE REVIEW

File: src/views/Home/FAQs/index.view.tsx [hunk 1]
```diff
@@ -8,71 +8,132 @@ const FAQs: React.FC = () => {
...
+              href='https://forms.gle/R9KtP8VMGGbfVTBx9'
+            >
+              {" "}
+              here
+            </a>{" "}
+            to apply.
+          </QaBlurb>
+          <QaBlurb question='What are CruzPoints?'>
+            CruzPoints is CruzHacks 2023’s point reward system for Hackers! For
+            more information,
```
Comment (Alperdec) [inline src/views/Home/FAQs/index.view.tsx]: We need a space here after information,

**Review by Alperdec — CHANGES_REQUESTED** [review]
Just one requested change with the text. The Cruzpoints info just needs a whitespace after the final comma and link.

This isn't necessary right now but, I think we should revisit this on a refactor and decouple the text from the `<QaBlurb>` component and insert them in a loop.

**Comment by Alperdec** [comment]
image

Also it appears that the svg icon is having some issues. Is this because the sizing is relative to the container?

**Comment by Alperdec** [comment]
Mobile view is spot on. One requested change and one buggy display

**Comment by zacktraczyk** [comment]
I was thinking about putting the QaBlurbs in a loop too.

**Comment by Alperdec** [comment]
I personally prefer it that way because it's easier to get lost in the sauce with too many items in a single component render. But like I said it's not a high priority currently

**Review by sidhantas — APPROVED** [review]
LGTM

**Review by Alperdec — APPROVED** [review]
Looks great

**Pull request merged**